.gitignore

```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+test.md
```

Comment:
As said by the previous TSE the test.md file should be ignored but I am still seeing it amongst your files. Please fix that.